packages/forms-web-app/src/routes/projects/documents.js

```diff
@@ -6,7 +6,7 @@ const aboutTheApplicationController = require('../../controllers/projects/docume
 
 router.get('/:case_ref', projectsController.getExamination);
 router.get('/:case_ref/documents/:page', aboutTheApplicationController.getAboutTheApplication);
-router.post('/:case_ref/documents/search/:page', aboutTheApplicationController.postSearchDocument);
-router.post('/:case_ref/documents/filter/:page', aboutTheApplicationController.postFilterDocument);
+router.post('/:case_ref/documents/:page', aboutTheApplicationController.postSearchDocument);
```

Comment:
We need the route to be navigable and shareable so let's just use the existing `get` route and pass in a query parameter for the search term

Reply:
Will do overall separate implementation once Elvin's code is merged, Would be easier to do all straight away. Its POST based on current implementation and can be changed to GET once Elvin's code is ready.